freqtrade/commands/cli_options.py

```diff
@@ -195,6 +195,36 @@ def __init__(self, *args, **kwargs):
         metavar='INT',
         default=constants.HYPEROPT_EPOCH,
     ),
+    "effort": Arg(
+        '--effort',
+        help=('The higher the number, the longer will be the search if'
+              'no epochs are defined (default: %(default)d).'),
+        type=float,
+        metavar='FLOAT',
+        default=constants.HYPEROPT_EFFORT,
+    ),
+    "mode": Arg(
+        '--mode',
+        help='Switches hyperopt to use one optimizer per job, use it'
+             'when backtesting iterations are cheap (default: %(default)s).',
+        metavar='NAME',
+        default=constants.HYPEROPT_MODE),
+    "n_points": Arg(
+        '--n-points',
```

Comment:
Whenever i give this a value of > 1 , i get `RuntimeError: Random evaluations exhausted and no model has been fit.`.

Will need investigation.
*note*: Seems to only happen in combination with `--mode multi`.